code/modules/mob/mob.dm

```diff
@@ -279,11 +279,25 @@
 		if(P.buckled || locate(/mob) in P.contents)
 			. += P.slowdown
 
-/mob/proc/Life()
+/**
+ * Handles the biological and general over-time processes of the mob.
+ *
+ *
+ * Arguments:
+ * - seconds_per_tick: The amount of time that has elapsed since this last fired
+ * - times_fired: The number of times SSmobs has fired
+ */
+/mob/proc/Life(seconds_per_tick = SSMOBS_DT, times_fired)
+	SHOULD_NOT_SLEEP(TRUE)
+	SHOULD_CALL_PARENT(TRUE)
+
 	if(LAZYLEN(spell_masters))
 		for(var/obj/screen/movable/spell_master/spell_master in spell_masters)
 			spell_master.update_spells(0, src)
 
+	if(stat != DEAD)
+		return TRUE
```

Comment:
The meaning of the returned value should be documented.

Reply:
It was supposed to be TRUE for "yes i am alive and processing" and FALSE or null for "i'm dead" but the Life procs are too inconsistent for it to be reliable, and I'm not rewriting them all to be at the moment

Note that this isn't different than other implementations of Life() that are already in the codebase, where they return TRUE at the end of it and null or FALSE otherwise, see eg. `/mob/living/simple_animal/Life` that returns a straight `1`